ui/raidboss/popup-text.js

```diff
@@ -59,6 +76,22 @@ class PopupText {
   }
 
   OnPlayerChange(e) {
+    // allow override of player via query parameter
+    // only apply override if player is in party
+    if (Options.PlayerNameOverride !== null &&
+        this.partyTracker.inParty(Options.PlayerNameOverride)) {
```

Comment:
Can you move this partyTracker check just before the `partyTracker.jobName` call? I had suggested the "job override" as a way to have this work when the party tracker doesn't have the person in the party.

Sorry for the comments here, but I just also filed #1153 which has issues with the party name.  So, this feels extra fragile.

Reply:
This has been updated in the latest changes. Please re-review this to make sure I understood your request properly.